www/src/App.js

```diff
@@ -7,7 +7,9 @@ import { Nav } from "./nav";
 import { Spec, Stats } from "./spec";
 import { Section } from "./section";
 import { Link } from "./link";
+import { List } from "./list";
 import specifications from "./result";
+import { Stats as StatsClass } from "./stats_class";
```

Comment:
nit: Why does the name need `Class` at the end?

Reply:
Line 7 import `Stats` from `"./spec"`